sda/cmd/finalize/finalize.go

```diff
@@ -96,12 +96,24 @@ func main() {
 
 				continue
 			}
-			if status == "disabled" {
+
```

Comment:
Should we really have special handling for `disabled`? They could still be reenabled by a message that hasn't been processed yet?

I do see why we wouldn't want to process the same message over and over, but are we sure that's not a use case we need to handle?

Reply:
The `disabled` case is due to the async handling in FEGA where we could get a cancellation message for a file that was currently being ingested.

Reply:
I believe I understand the context but see a (possible although unlikely) similar race condition between whatever message/service is responsible for changing the state `disabled` -> `enabled` and the message for finalize.

But I also see it as low risk, so probably fine to (consciously) ignore.